### PR TITLE
Development section

### PR DESCRIPTION
**Pull request by onishiweb**

### CODE REVIEW

File: playbook.md [hunk 1]
```diff
@@ -106,29 +106,56 @@ If you are making a substantial change (perhaps following a team conversation) p
 
 ### Sprinting
 
-* planning and retros (dev perspective)
-* design and dev
-* stories and points
-* backlog, prioritisation
+Our sprints start and end with planning and retrospective sessions. Both sessions involve all members of the project team (developers, user researcher, and delivery manager) and the product owner from the client side - retro’s may sometimes include more of the client team if they are doing a show and tell.
+
+The sprint planning session is used to decide on stories for the upcoming sprint. Stories are discussed with the client and based on user needs of the project. Developers will then estimate the stories with a number of points relating to the complexity of the story, using the numbers of the [Fibonacci sequence](http://www.mathsisfun.com/numbers/fibonacci-sequence.html). This is usually done through a game of [Planning Poker](https://en.wikipedia.org/wiki/Planning_poker) between the developers on the project. Once estimated, stories are prioritised by the client and put into the backlog for the sprint. Developers should also advise at this point if there are any dependancies the client should be aware of.
```
Comment (dgmstuart) [inline playbook.md]: * Can you break up this paragraph into shorter chunks? Maybe into stories, estimation and prioritisation?
* I think that use of fibonacci is a team decision, not a company policy.

File: playbook.md [hunk 2]
```diff
@@ -106,29 +106,56 @@ If you are making a substantial change (perhaps following a team conversation) p
...
-  * Purpose
-  * Messages
-  * Granularity
-* Pull requests and code review
-  * Make PRs when done
-  * do code reviews - scope, purpose
-* Deploying
+To start any new piece of work - whether story, bug fix, or chore - you should always start by creating a new branch from the `master` branch of the main repository on Github. (If the main repository is not yet on Github, [move it there](https://github.com/dxw/playbook/blob/master/guides/moving-a-repo-to-github.md)). 
+
+We follow the [Git ENV](https://www.wearefine.com/mingle/env-branching-with-git/) model for branching using the [Git Env Tool](https://github.com/dxw/git-env). Follow the [Git Env Guide](https://github.com/dxw/playbook/blob/master/guides/git-env.md) to get up to speed on working with Git ENV. Branches created should be consistently named, based on either of the following patterns:
```
Comment (dgmstuart) [inline playbook.md]: TVH team, don't (and can't) follow git env (or indeed git-flow) because of our deployment pipeline

File: playbook.md [hunk 3]
```diff
@@ -106,29 +106,56 @@ If you are making a substantial change (perhaps following a team conversation) p
...
-  * follow the style of the thing you're in
-* Committing
-  * Purpose
-  * Messages
-  * Granularity
-* Pull requests and code review
-  * Make PRs when done
-  * do code reviews - scope, purpose
-* Deploying
+To start any new piece of work - whether story, bug fix, or chore - you should always start by creating a new branch from the `master` branch of the main repository on Github. (If the main repository is not yet on Github, [move it there](https://github.com/dxw/playbook/blob/master/guides/moving-a-repo-to-github.md)).
```
Comment (dgmstuart) [inline playbook.md]: TVH team use separate master/develop branches

File: playbook.md [hunk 4]
```diff
@@ -106,29 +106,56 @@ If you are making a substantial change (perhaps following a team conversation) p
...
+Pull requests should be made when a feature is completed by a developer at the same time that it is passed to a delivery manager to review. Once the reviewing developer, delivery manager, and client are happy with the feature, it can then be merged with the `master` branch and deployed to production.
+
+When performing a code review, you should look to make sure:
+* The chosen solution is appropriate
+* Code is written in a way that is easy to understand or explained via relevant comments
+* Code follows our [code style guides](#code-style)
+
+#### Deploying
+
+We maintain two environments for deployments, `testing` and `production`. Deployments to `testing` should be done throughout the sprint ready for sign off from the client. Deployments to `production` should be made once code has been signed off from the client and passed a code review from another developer.
```
Comment (dgmstuart) [inline playbook.md]: Isn't this only true of GovPress projects?

File: playbook.md [hunk 5]
```diff
@@ -106,29 +106,56 @@ If you are making a substantial change (perhaps following a team conversation) p
 
 ### Sprinting
 
-* planning and retros (dev perspective)
-* design and dev
-* stories and points
-* backlog, prioritisation
+Our sprints start and end with planning and retrospective sessions. Both sessions involve all members of the project team (developers, user researcher, and delivery manager) and the product owner from the client side - retro’s may sometimes include more of the client team if they are doing a show and tell.
```
Comment (dgmstuart) [inline playbook.md]: retros not retro's

Comment (onishiweb) [inline playbook.md]: Fixed

File: playbook.md (hunk 2 above)
Comment (harry-m) [inline playbook.md]: :+1:

File: playbook.md (hunk 1 above)
Comment (harry-m) [inline playbook.md]: I'll add an issue for the former and a caveat for the latter.

File: playbook.md (hunk 4 above)
Comment (harry-m) [inline playbook.md]: Yes, but that's everything except TVH. I don't want to introduce too much ambiguity. Might need a section at the top for "big projects set their own rules"?

**Pull request merged**